README.md

```diff
@@ -37,7 +61,15 @@ A [tutorial](https://github.com/bids-standard/bids-matlab/blob/master/examples/t
 
 ## Requirements
 
-BIDS-MATLAB works with MATLAB R2014a or newer, or Octave 4.2.2 or newer. (It may also work with older versions, but those are not actively supported.)
+BIDS-MATLAB works with: 
+
+- Octave 5.2.0 or newer
+- MATLAB R2014a or newer  
+
+We aim for compatibility with the latest stable release of Octave at any time. Compatibility can sometimes also be achieved with older versions of Octave but this is not guaranteed.
```

Comment:
```suggestion
We aim for compatibility with the latest stable major version of Octave at any time. Compatibility can sometimes also be achieved with older versions of Octave but this is not guaranteed.
```

Maybe you should declare support for a full major version series like Octave 5.x instead of just the "latest stable release", which could be interpreted as "exactly 5.2.3" or similar.

Maybe declare a policy about how long Octave major versions are supported? For example, Octave 6.x is right around the corner. How long will you support Octave 5.x once 6.x is out?

And what about a policy for how long older Matlab versions are supported? If you support R2014a forever, you're eventually going to have a _lot_ of Matlab versions to test against, and you'll never get to adopt newer Matlab features.